### ISSUE TITLE
Pull Request #38 Failing

### ISSUE THREAD

**Issue by philkim72** [jtr13/cc22mw#62]
Pull Request #38 is failing.  I have removed geocomputationwithr.Rmd from the bookdown file for now.  

Action Failure: https://github.com/jtr13/cc22mw/actions/runs/3484783948
Detailed run message:  https://github.com/jtr13/cc22mw/actions/runs/3484783948/jobs/5829693461

**Comment by jtr13** [comment]
It seems that the **leaflet** package is causing this problem: https://github.com/jtr13/cc22tt/issues/24

I would recommend setting `eval=FALSE` for any chunks that use **leaflet** and add a comment `# not run` to the chunks.

**Comment by philkim72** [comment]
Can you clarify how to set eval=False or can you tell me if what I did below is correct?

```{r, eval=False} 
leaflet()%>%
  addProviderTiles("NASAGIBS.ViirsEarthAtNight2012") %>%
  addMarkers(
    lng = 73, lat = 18,
    label = "Home Sweet Home",
    labelOptions = labelOptions(noHide = T))%>%
  addMarkers(
    lng = -85, lat = 31,
    label = "Columbia University",
    labelOptions = labelOptions(noHide = T))%>%
  addMarkers(
    lng = -19, lat = 64,
    label = "Iceland - Dream Destination",
    labelOptions = labelOptions(noHide = T))
```

**Comment by jtr13** [comment]
The simplest thing to do is globally set eval to FALSE for all the code chunks by changing this line (near the top):

```
knitr::opts_chunk$set(warning = FALSE, message = FALSE)
```

to 

```
knitr::opts_chunk$set(warning = FALSE, message = FALSE, eval = FALSE)
```

**Comment by philkim72** [comment]
Thank you for the clarification.  So setting eval = FALSE globally prevents the r chunks from running.  What are we supposed to do next?

**Comment by jtr13** [comment]
After doing that you can uncomment the chapter in `_bookdown.yml`

**Comment by philkim72** [comment]
Yes, I already did that.  But doesn't that make their page inoperable?

**Comment by jtr13** [comment]
Yes, unfortunately that's the best option we have :-/ .
